docs/msbuild/usingtask-element-msbuild.md

```diff
@@ -34,6 +34,9 @@ Maps the task that is referenced in a [Task](../msbuild/task-element-msbuild.md)
     Condition="'String A'=='String B'" />
 ```
 
+> [!NOTE]
+> Unlike properties and items, the *first* `UsingTask` element that applies to a `TaskName` will be used; to override tasks you must define a new `UsingTask` *before* the existing one.
```

Comment:
@Mikejo5000 or @rainersigwald  Can you review this change in this MSBuild file?

Reply:
That looks like it should have come from MicrosoftDocs/visualstudio-docs#4040; it looks ok, but I don't understand why it's showing up here.